CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * Pass Python interpreter information to driver runtimes that support it. This is used by NI in order to better understand client usage.
```

Comment:
Until we update the metadata for the other drivers, it only applies to NI-DCPower.

Reply:
Actually, the comment doesn't apply to this PR until we are actually passing the Python interpreter information to driver runtimes. Adding the internal function but not calling doesn't need to be mentioned in the CHANGELOG.md since it has no functional implications for the customer.

Once we are passing the information, then the comment will apply and will apply under the ALL section.